### DISCUSSION TITLE
HFS for Synology and Open Media Vault NAS Systems

### DISCUSSION THREAD

**Discussion by Messtechniker** [rejetto/hfs#488] [Ideas]
Just an Idea: 
Since as far as I know HFS3 runs on Node.js 18 and since Synology NAS software and
Open Media Vault NAS possibly via Docker are seemingly compatible,
porting of HFS3 might be a realistic possibility. How much work this 
would involve, I don't know.
Just pointing to you to a possibility. Not a request.

**Comment by PH5HP** [comment]
Hi,

Or even running within True NAS :-) ..

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i know someone was working on hfs+docker.
i'm not planning to work on it for the moment, but i can give support

**Comment by Messtechniker** [comment]
Thanks for the hint. Will try and see what going on with HFS docker 
in connection with Open Media Vault.

**Comment by damienzonly** [comment]
Hi @Messtechniker, i just released official docker images of hfs https://hub.docker.com/r/rejetto/hfs, let me know here for any issue https://github.com/damienzonly/hfs-docker/issues
